Tests/TensorFlowTests/OperatorTests/MathTests.swift

```diff
@@ -496,6 +576,16 @@ final class MathOperatorTests: XCTestCase {
     }
 
     static var allTests = [
+        ("testAbs", testAbs),
```

Comment:
It's generally a good idea to keep the function declarations above in the same order as this list. So, I would move your newly added `testAbs`-`testIdentity` after `testElementaryFunctions`.

Reply:
Will do!